lib/Dialect/Substrait/IR/Substrait.cpp

```diff
@@ -155,6 +155,22 @@ EmitOp::inferReturnTypes(MLIRContext *context, std::optional<Location> loc,
   return success();
 }
 
+LogicalResult
+FetchOp::inferReturnTypes(MLIRContext *context, std::optional<Location> loc,
+                          ValueRange operands, DictionaryAttr attributes,
+                          OpaqueProperties properties, RegionRange regions,
+                          llvm::SmallVectorImpl<Type> &inferredReturnTypes) {
+  Value input = operands[0];
+
+  TypeRange leftFieldTypes = cast<TupleType>(input.getType()).getTypes();
+
+  auto resultType = TupleType::get(context, leftFieldTypes);
+
+  inferredReturnTypes = SmallVector<Type>{resultType};
+
```

Comment:
Nit: there are too many blank lines here. I'd say you don't need any of them; it'll still be just five and all pretty straightforward...